### PR TITLE
Djibouti

### PR DESCRIPTION
**Pull request by aaron-kaplan**
Sorry, I thought I had merged this without review while you were away, but in fact it was just sitting in an unpushed branch. We don't have the datasets yet, but you can use this to test the shapes.

### CODE REVIEW

**Comment by aaron-kaplan** [comment]
I added the icon to dlconfig already: https://bitbucket.org/iridl/dlconfig/commits/804e1ae

**Comment by aaron-kaplan** [comment]
@kgraaf I pushed a new commit that adds the rain-djibouti dataset. I've already installed the dataset in /data/aaron/fbf.

We've gone through two iterations with Kyle on the forecast dataset and he's working on a third.

File: fbfmaproom/fbfmaproom-sample.yaml [hunk 1]
```diff
@@ -454,3 +454,62 @@ countries:
...
+              vuln_sql: select adm0_code as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Djibouti' group by 1, 2
+
+            - name: Regional
+              sql: select code as key, djibouti_adm01 as label,
+                  ST_AsBinary(the_geom) as the_geom from iridb.djibouti_adm01
+              vuln_sql: select code as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Djibouti' group by 1, 2
+
+            - name: District
```
Comment (aaron-kaplan) [inline fbfmaproom/fbfmaproom-sample.yaml:479]: Should this be renamed Sub-region?

Comment (kgraaf) [reply fbfmaproom/fbfmaproom-sample.yaml:479]: I'm honestly not sure. Liban seemed to suggest to me that Subregion is correct but was inconsistent there.

**Comment by aaron-kaplan** [comment]
@kgraaf and I finished this branch together.

**Comment by aaron-kaplan** [comment]
The forecast doesn't look right, but fixing that shouldn't require any changes to the code or configuration.

**Pull request merged**